flink-cdc-e2e-tests/flink-cdc-source-e2e-tests/src/test/java/org/apache/flink/cdc/connectors/tests/utils/FlinkContainerTestEnvironment.java

```diff
@@ -119,17 +117,16 @@ public abstract class FlinkContainerTestEnvironment extends TestLogger {
     private GenericContainer<?> jobManager;
     private GenericContainer<?> taskManager;
 
-    @Parameterized.Parameters(name = "flinkVersion: {0}")
-    public static List<String> getFlinkVersion() {
+    public static String getFlinkVersion() {
         String flinkVersion = System.getProperty("specifiedFlinkVersion");
-        if (flinkVersion != null) {
-            return Collections.singletonList(flinkVersion);
-        } else {
-            return Arrays.asList("1.19.1", "1.20.0");
+        if (Objects.isNull(flinkVersion)) {
+            throw new IllegalArgumentException(
+                    "No Flink version specified to run this test. Please use -DspecifiedFlinkVersion to pass one.");
```

Comment:
Previous test has default value. But it is good idea to throw here, since the release cycle is very frequent nowadays 👍